modules/querier/http.go

```diff
@@ -99,36 +104,40 @@ func (q *Querier) TraceByIDHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 // return values are (blockStart, blockEnd, queryMode, error)
-func validateAndSanitizeRequest(r *http.Request) (string, string, string, error) {
+func validateAndSanitizeRequest(r *http.Request) (string, string, string, int64, int64, error) {
```

Comment:
These new params also need to be checked on the query frontend and rejected in the event that they are invalid. 

We are currently checking the trace id here: https://github.com/grafana/tempo/blob/main/modules/frontend/frontend.go#L10

Perhaps we should move `validateAndSanitizeRequest` to the `/pkg/api` package? Then we could use the same method to check the params in both places?